man/systemd.netdev.xml

```diff
@@ -1070,6 +1070,33 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ActiveSlave=</varname></term>
+        <listitem>
+          <para>A boolean. Specifies the new active slave for modes that support it.
+          <literal>active-backup</literal>,
+          <literal>balance-alb</literal> and
+          <literal>balance-tlb</literal>. Defaults to false.
```

Comment:
I don't grok this. the first sentence and the last sentence suggest this is a boolean, but in the middle you list enum values?